cypress/integration/boards.spec.js

```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
+        cy.get(boards.checkOrgFromList).eq(1).click();
         cy.get(boards.nextBtn).click();
         cy.get(boards.checkScrumRb).click();
         cy.get(boards.nextBtn).click();
         cy.get(boards.nextBtn).click();
         cy.get(boards.nextBtn).click();
         cy.get(boards.nextBtn).click();
-    })
 
-    it("Check board exist", () => {
-        cy.get("ul[class='vs-c-list vs-c-list--boards']").should('contain', dataBoard.board.brdName)
+        cy.get('li[class="vs-c-list__item has-caret"]').should(($loggedInUser) => {
+            expect($loggedInUser).to.contain(dataBoard.board.brdName)
+        })
     })
 
-    it("Config board", () => {
-        cy.get(boards.activeBoard).eq(0).click();
+    it("Edit board code", () => {
+        cy.get(boards.activeBoard).eq(3).click();
         cy.get(boards.editBoard).eq(8).click();
+        cy.get(boards.configBoardCode).clear();
+        cy.get(boards.configBoardCode).type("bb{enter}");
+
+        cy.get('.el-message__group > p')
+            .should('be.visible')
+            .and('contain', 'Successfully updated the Board Basic Info.')
     })
 
     it("Edit board name", () => {
-        cy.get(boards.activeBoard).eq(0).click();
+        cy.get(boards.activeBoard).eq(3).click();
         cy.get(boards.editBoard).eq(8).click();
+        cy.get(boards.edinBrdName).eq(0).clear()
         cy.get(boards.edinBrdName).eq(0).type(dataBoard.board.editBrdName);
         cy.get(boards.saveUpdate).eq(0).click()
+
+        cy.get('.el-message__group > p')
+            .should('be.visible')
+            .and('contain', 'Successfully updated the Board Basic Info.')
     })
 
-    it("Delete board", () => {
-        cy.get(boards.activeBoard).eq(0).click();
+    it.only("Delete board", () => {
+        cy.get(boards.activeBoard).eq(3).click();
         cy.get(boards.editBoard).eq(8).click({ force: true });
         cy.get(boards.deleteBoard).click()
         cy.get(boards.confirmDeleteBrd).click()
+
+        cy.get('[class="vs-c-organization-boards"]').should(($board) => {
+            expect($board).to.contain("testB")
+        })
+    })
+
+    //negative
+    it("Edit board code with 5 characters", () => {
+        cy.get(boards.activeBoard).eq(3).click();
+        cy.get(boards.editBoard).eq(8).click();
+        cy.get(boards.configBoardCode).type("board code change{enter}");
+
+        cy.get('.el-form-item__error el-form-item-error--top').should(($editBoard) => {
+            expect($editBoard).to.contain("The board code field may not be greater than 4 characters")
+        })
+    })
+
+    it("Edit board without name", () => {
+        cy.get(boards.activeBoard).eq(3).click();
+        cy.get(boards.editBoard).eq(8).click();
+        cy.get(boards.edinBrdName).eq(0).clear()
+
+        cy.get("button[type='submit']").should('be.disabled')
+        cy.get("span[class='el-form-item__error el-form-item-error--top']").should('contain', "The board title field is required")
```

Comment:
Should also probably be stored in some JSON file with selectors

Reply:
103 and 104 are asserted. Should need in the JSON file?